docs/cli/dotnet-publish.md

```diff
@@ -0,0 +1,72 @@
+dotnet-publish
+==============
+
+## NAME
+
+`dotnet-publish` - packs the application and all of its dependencies into a folder getting it ready for publishing
+
+## SYNOPSIS
+
+dotnet-publish [--framework]  
+    [--runtime] [--output]  
+    [--configuration]  
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-publish` compiles the application, reads through its dependencies specified in `project.json` and publishes the resulting set of files to a directory. 
+
+Depending on the type of portable app, the directory contains the following:
+
+1. **Portable application** - application's intermediate language (IL) code and all of application's managed dependencies.
+    * **Portable application with native dependencies** - as above with a sub-directory for each native dependencies' 
+    supported platform. 
+2. **Self-contained application** - as above as well as the entire runtime for the targeted platform.
```

Comment:
As of right now that app must be netstandardapp1.5 (soon to be netcoreapp1.0) to be a self-contained application.  If it is not mentioned here, it should be mentioned above about self-contained apps needs to be *app#.#.